homeassistant/components/go2rtc/__init__.py

```diff
@@ -81,6 +89,10 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     ):
         if not is_docker_env():
             if not configured_by_user:
+                # Remove config entry if it exists
+                if entries := hass.config_entries.async_entries(DOMAIN):
+                    for entry in entries:
+                        await hass.config_entries.async_remove(entry.entry_id)
```

Comment:
Why do we need to remove the config entry?
Can we not adjust the config entry that it will be updated instead

Reply:
I don't agree, if there's no user configuration anymore and we're not on docker, there's no need for a config entry